api/src/wfl/util.clj

```diff
@@ -168,22 +168,6 @@
   [f m]
   (zipmap (keys m) (map f (vals m))))
 
-(defn minutes-between
```

Comment:
I noticed that these two utils aren't used (and the `minutes-between` actually has a bug, it's returning the difference in seconds) so I removed.  I can re-add and fix if they're helpful.